client/components/date-picker/day.jsx

```diff
@@ -1,87 +1,118 @@
 /**
  * External Dependencies
  */
-var React = require( 'react' ),
-	noop = require( 'lodash/noop' ),
-	classNames = require( 'classnames' );
+import React, { PropTypes, Component } from 'react';
+import { noop, map } from 'lodash';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-var Tooltip = require( 'components/tooltip' );
-
-module.exports = React.createClass( {
-	displayName: 'DatePickerDay',
-
-	propTypes: {
-		date: React.PropTypes.object.isRequired,
-		events: React.PropTypes.array
-	},
-
-	getInitialState: function() {
-		return {
-			showTooltip: false
-		};
-	},
-
-	isPastDay: function( date ) {
-		var today = this.moment().set( {
+import Tooltip from 'components/tooltip';
+import { CalendarEvent } from './event';
+
+class DatePickerDay extends Component {
+	static propTypes = {
+		date: PropTypes.object.isRequired,
+		events: PropTypes.array,
+		moment: PropTypes.func,
+		maxEventsPerTooltip: PropTypes.number,
+	};
+
+	static defaultProps = {
+		maxEventsPerTooltip: 8,
+	};
+
+	state = {
+		showTooltip: false,
+	};
+
+	isPastDay( date ) {
+		const today = this.props.moment().set( {
 			hour: 0,
 			minute: 0,
 			second: 0,
 			millisecond: 0
 		} );
 
 		date = date || this.props.date;
-
 		return ( +today - 1 ) >= +date;
-	},
+	}
 
-	handleTooltip: function( show ) {
-		var showTooltip = ! ! this.props.events.length && show;
-		this.setState( { showTooltip: showTooltip } );
-	},
+	showTooltip = () => {
+		if ( ! this.props.events.length ) {
+			return;
+		}
 
-	renderTooltip: function() {
-		var label;
+		this.setState( { showTooltip: true } );
+	}
+
+	hideTooltip = () => {
+		this.setState( { showTooltip: false } );
+	}
 
+	renderTooltip() {
 		if ( ! this.state.showTooltip ) {
-			return;
+			return null;
 		}
 
-		label = this.translate(
-				'%(posts)d post',
-				'%(posts)d posts', {
-					count: this.props.events.length,
-					args: {
-						posts: this.props.events.length
-					}
-				}
-			);
+		if ( ! this.props.events.length ) {
+			return null;
+		}
+
+		const label = this.props.translate(
+			'%d Post',
```

Comment:
Suggestion is to use lowercase, if that makes sense here.
The 2nd option in the PR translation status page is to use the named placeholder (also lowercase), like in [here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/stats/post-trends/day.jsx#L45).